src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -619,7 +639,7 @@ private boolean nodesReadyForNewRepair(
 
     Collection<String> nodes = getNodesInvolvedInSegment(dcByNode);
     String dc = EndpointSnitchInfoProxy.create(coordinator).getDataCenter();
-    boolean requireAllHostMetrics = DatacenterAvailability.ALL == context.config.getDatacenterAvailability();
+    boolean requireAllHostMetrics = DatacenterAvailability.LOCAL != context.config.getDatacenterAvailability();
```

Comment:
isn't this changing the way `EACH` works? and isn't that outside of the scope of this work?

Reply:
Not as far as I can tell.
`EACH` always required to have all host metrics in order to proceed with the segment. `LOCAL`  is the only mode where all hosts metrics aren't required (just those from the local DC).

Reply:
Cool, thanks. (The check against`allLocalDcHostsChecked` is done in the `okToRepairSegment(..)` method.)